### PR TITLE
postgrator upgrade

### LINKED ISSUE #755: Update postgrator
**Describe the problem/challenge you have**

[A description of the current limitation/problem/challenge that you are experiencing.]

**Describe the solution you'd like**

[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Retraced version:
- SDK:

### PR DESCRIPTION
**Pull request by ukrocks007**
Thank you for contributing to Retraced!

# Upgrade postgrator

# Does your change fix a particular issue?

Fixes #755

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "pg-format": "1.0.4",
     "picocolors": "1.0.0",
     "pino": "8.8.0",
-    "postgrator": "3.0.0",
+    "postgrator": "5.0.1",
```
Comment (deepakprabhakara) [inline package.json:38]: Isn't 7.7.1 the latest version?

Comment (ukrocks007) [reply package.json:38]: That is ESM. I tried it gave some errors that can not import. So I started to go up iteratively.

Comment (deepakprabhakara) [reply package.json]: Sounds good

File: src/models/saved_export/render.ts [hunk 2]
```diff
@@ -10,8 +10,9 @@ import { parseQuery, ParsedQuery } from "../event";
 import QueryDescriptor from "../query_desc/def";
 import { logger } from "../../logger";
 import config from "../../config";
+import { stringify } from "csv-stringify";
 
-let stringify;
+// let stringify;
```
Comment (deepakprabhakara) [inline src/models/saved_export/render.ts:15]: this should be cleaned up

**Review by deepakprabhakara — APPROVED** [review]

**Pull request merged**